priv/repo/migrations/20170517092511_create_attendance.exs

```diff
@@ -0,0 +1,16 @@
+defmodule CoursePlanner.Repo.Migrations.CreateAttendance do
+  use Ecto.Migration
+
+  def change do
+    create table(:attendances) do
+      add :attendance_type, :string
```

Comment:
so this is a tricky question.
cause we can put this option to setting so client can change it, or we can hardcode them